src/repository/transform_photos.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from typing import Type
+
+import cloudinary
+from cloudinary import uploader
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+import uuid
+
+from src.conf.config import settings
+from src.database.models import Photo
+from src.repository.photos import get_photo_by_photo_id
+from src.schemas import TransformPhotoModel
+
+
+async def update_orig_photo_with_transformed_photo(
+    db: Session, photo_id: int, photo_url: str, updated_by: int, photo_description: str
+):
+    """
+    Method updates the original photo in the db with a transformed photo.
+
+    :param db: DB instance.
+    :type db: Session.
+    :param photo_id: Original photo identifier.
+    :type photo_id: int.
+    :param photo_url: Original photo URL.
+    :type photo_url: str.
+    :param updated_by: User who updated a photo.
+    :type updated_by: int.
+    :param photo_description: Photo description.
+    :type photo_description: str.
+    :return: Photo instance.
+    :rtype: Photo.
+    """
+    original_photo: Type[Photo] | None = await get_photo_by_photo_id(
+        db=db, photo_id=photo_id
+    )
+    if not original_photo:
```

Comment:
Better to keep logic in one place - routes path and here to return None instead of exception, since the 'repository' partition in usual way is not responsible for business logic, just for data interaction according to our guru GPT.